README.md

```diff
@@ -185,7 +185,9 @@ roslaunch slam_simulations file_name.launch
                      of the robot's surroundings.
 
  - navigation_stack.launch - used to autonomously navigate the bot in the environment
-
+ 
```

Comment:
Instead of mentioning about darknet_ros in the 5th point, create a 6th point and first mention about Darknet (The current description which you have provided is enough), then provide them the link to the repo and ask them to follow the instructions mentioned in the repo for cloning and building it and then write the full command to launch the darknet_ros.launch file in a code block.